package.json

```diff
@@ -48,12 +48,12 @@
     "addons-scanner-utils": "5.0.0",
     "ajv": "6.12.6",
     "ajv-merge-patch": "4.1.0",
+    "async": "3.2.2",
```

Comment:
We add a few new deps here, which isn't worse than before but maybe we could split the code of the `Dispensary` class to avoid this problem. I don't know "how" exactly, though.